airflow/www/templates/airflow/dag.html

```diff
@@ -145,11 +145,20 @@ <h4 class="modal-title" id="myModalLabel">
                 <input name="task_id" type="hidden">
                 <input name="execution_date" type="hidden">
                 <input name="origin" type="hidden" value="{{ request.base_url }}">
-                <button id="btn_run_from_here" type="button" class="btn btn-primary"
+                <table>
+                  <tr>
+                  <td>
+                    <button id="btn_run_from_here" type="button" class="btn btn-primary"
                         title="Run this task and all downstream tasks"
                         data-action="{{ url_for('airflow.run_from_here') }}">
-                  Retry This Task and Run All Downstream
-                </button>
+                  Retry This Task and Run All Downstream Tasks
```

Comment:
This should be now renamed to:
```suggestion
                  Retry This Task
```
Since "Include Downstream DAGs?" is already a checkbox below

Reply:
updated